draft-ietf-pquip-pqc-engineers.md

```diff
@@ -360,7 +360,7 @@ Stateful hashbased signatures with long service lifetimes require additional ope
 
 The SLH-DSA algorithm on the other hand leverages the HORST (Hash to Obtain Random Subset with Trees) technique and remains the only hash based signature scheme that is stateless, thus avoiding all the complexities with state management.
 
-SLH-DSA is an advancement on SLH-DSA which reduces the signature sizes in SLH-DSA and makes it more compact. SLH-DSA was recently standardized by NIST.
+SLH-DSA is an advancement on SPHINCS which reduces the signature sizes in SPHINCS and makes it more compact. SLH-DSA was recently standardized by NIST.
```

Comment:
I would re-phrase the text as follows:
SLH-DSA is an advancement over SPHINCS that reduces the signature sizes and makes them more compact.